mongo/log_monitors.go

```diff
@@ -10,46 +10,46 @@ import (
 	"context"
 
 	"go.mongodb.org/mongo-driver/event"
-	"go.mongodb.org/mongo-driver/mongo/mongologger"
+	"go.mongodb.org/mongo-driver/mongo/mongolog"
 )
 
-func newCommandLogger(l *mongologger.MongoLogger, cm *event.CommandMonitor) *event.CommandMonitor {
+func newCommandLoggingMonitor(logger *mongolog.MongoLogger, cm *event.CommandMonitor) *event.CommandMonitor {
```

Comment:
nit: Maybe `newLoggingCommandMonitor` to keep the words `CommandMonitor` together?

Also, this should have some documentation either here or in the `configure` function (or both?) to describe what we're doing because it might not be obvious to a new reader.

Reply:
done. added a description here